.github/workflows/create-release-pr.yaml

```diff
@@ -0,0 +1,28 @@
+name: "Create release-please PR"
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    outputs:
+      release_created:
+        description: "If true, a release PR has been merged"
+        value: ${{ jobs.release-please.outputs.release_created }}
+      tag_name:
+        description: "The release tag. Ex v1.4.0"
+        value: ${{ jobs.release-please.outputs.tag_name }}
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          release-type: simple
+          package-name: "dm-cli"
+          extra-files: |
+            requirements.txt
```

Comment:
The extra-files should be used to list files that should be updated with the newest version in each release-please PR. For release-please to find the correct line, you must comment it with `# x-release-please-version`. Some files are added by default, depending on which release-type you're using, and in those cases I don't think you need the comment.

I think that if you set release-type to `python`, it will automatically update the dm_cli/__init__.py and setup.py files with the correct version number. I don't think there are any more files that need updating, so the extra-files input could probably be removed

Reply:
cool!